ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -139,7 +139,8 @@
         {:keys [spec-key spec-version provider-id native-id request-context concept]} res
         metadata (:metadata concept)]
     (validate-document-against-schema spec-key spec-version metadata)
-    (tgen/create-generic request-context [provider-id native-id] (json/generate-string concept))))
+    (tgen/create-generic request-context [provider-id native-id] (json/generate-string concept))
+    {:status 204}))
```

Comment:
Why don't we want the actual message coming back to the user that the concept was ingested?  Now all you are doing is providing nothing back. When ingesting collections you get back the actual collection with all its errors and warnings.
I think we should do the same.  That can be done by removing {:status 204}. I think the behavior should be consistent.

Reply:
I thought we wanted to return no content with read and update?

Reply:
What is the point of read if you don't get something back?  What does read do?  If there are errors or warnings, these should be surfaced back to the caller.

Reply:
I am OK, with creating a ticket and doing it later.